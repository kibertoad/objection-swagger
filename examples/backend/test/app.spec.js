const { getApp } = require('../app');
const request = require('supertest');

describe('example app', () => {
  it('returns Swagger document correctly', async () => {
    const app = await getApp();

    const response = await request(app).get('/api-docs');
    expect(response.body).toMatchSnapshot();
  });
});
