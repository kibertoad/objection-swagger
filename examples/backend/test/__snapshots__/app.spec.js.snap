// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example app returns Swagger document correctly 1`] = `
Object {
  "info": Object {
    "title": "Example application",
    "version": "1.0.0",
  },
  "paths": Object {
    "/health-check": Object {
      "get": Object {
        "description": "Check heartbeat",
        "produces": Array [
          "application/json",
        ],
        "responses": Object {
          "200": Object {
            "description": "heartbeat report",
          },
        },
      },
    },
  },
  "swagger": "2.0",
}
`;
